### ISSUE TITLE
Setting config.use_ember_middleware = false does not disable middleware in non-production environments 

### ISSUE THREAD

**Issue by bradurani** [rwz/ember-cli-rails#70]
Following the instructions, I do

```
#test.rb
 config.use_ember_middleware = false
```

yet the middleware is still loaded. Here's the boolean logic responsible

```ruby
#ember-cli-rails/lib/ember-cli/helpers.rb, line 28 
def use_middleware?
   rails_config_for(:use_ember_middleware) || non_production?  
end
```

My intuitive understanding is it should be something like
```ruby
def use_middleware?
   if(rails_config_defined?(:use_ember_middleware))
      rails_config_for(:use_ember_middleware)
  else
      non_production?
  end
end
```

Which one is the intended behavior?

**Comment by rwz** [comment]
You're absolutely right. Setting these config parameters should override defaults, not augment logic. It's a bug. Fixed by https://github.com/rwz/ember-cli-rails/commit/9fa822067daedf23a0871ca0d1bd8f9965e480dd

**Comment by rwz** [comment]
Fixed in 0.1.3
